tutorials/p5js/_posts/2017-10-14-review.md

```diff
@@ -82,7 +82,7 @@ Either way, the style page looks like this:
 
 {% include codepen.html slug-hash="MJWYNQ" height="qqjzvM" %}
 
-Remember that you can select elemnts by **class** or **ID**, and differnt styles cascade (in other words, combine) to decide the final styling of an element. Learn more about CSS [here](/tutorials/html/css) or on [W3SChools](https://www.w3schools.com/css/) or [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS).
+Remember that you can select elemnts by **class** or **ID**, and differnt styles cascade (in other words, combine) to decide the final styling of an element. Learn more about CSS at [my CSS tutorial]](/tutorials/html/css) or on [W3SChools](https://www.w3schools.com/css/) or [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS).
```

Comment:
Can we change "my CSS tutorial" to "the CSS tutorial" here?

I don't know why, but I prefer keeping the ownership a bit more general. The tutorials belong to the site, not to me, if that makes sense?